03-understanding-survey-data-files.Rmd

```diff
@@ -1 +1,238 @@
+
+## Types of survey documentation
+
+### Technical documentation
+<!-- this is adapted from chapter 05 ending -->
+The technical documentation, which may be called user guides or methodology/analysis guides, will highlight the variables necessary to specify the survey design. We recommend focusing on these key sections:
+
+* **Introduction:** The introduction orients us to the survey. Generally, this section provides the project's background, the study's purpose, and the main research questions.
+* **Study design:** The study design section describes how researchers prepared and administered the survey was prepared.
+* **Sample:** The sample section describes the sampling of the survey: how researchers selected cases, any sampling error that occurred, and the limitations of the sample. This section can contain recommendations on how to use sampling weights. Look for weight information, whether the survey design is strata and/or clusters/PSUs or replicate weights, and any population sizes or finite population correction. This documentation is critical in successfully running our analysis, and more detail on sample designs is available in [Chapter 05: Specifying sample designs in srvyr](#c05).
+
+In addition to survey design, the technical documentation may include other helpful information. Some technical documentation includes syntax for SAS, SUDAAN, Stata, and/or R, meaning we don't have to create this code from scratch. This saves us time and ensures our code is accurate.
+
+### Questionnaires
+
+A survey questionnaire is the series of questions asked to obtain information from survey respondents. A questionnaire may employ different types of questions, such as multiple-choice, open-ended, Likert scales, or ranking questions, to gather opinions, behaviors, or demographic data. The questionnaire may also randomize responses or include instructions to help the respondent understand the question. A survey may have one questionnaire or multiple, depending on its scale and scope.
+
+The questionnaire is an essential resource for understanding and interpreting the survey data, and should be used alongside any analysis. It provides details about each of the questions asked in the survey, such as question name, question wording, response options, question logic, randomizations, display specification, mode differences, and the universe (if only a subset of respondents were asked the question). 
+
+Below, we show a question from the ANES 2020 questionnaire. This part shows a particular question's question name (`postvote_rvote`), description (Did R Vote?), full wording of the question and responses, response order, universe, question logic (if `vote_pre` = 0), and other specifications. It also includes the variable name, which can be used to link to the codebook.
+
+![](images/questionnaire-example.jpg)
```

Comment:
I prefer images with captions and we need to add in alt-text. One way to do this:

````suggestion
Below in Figure \@ref(fig:que-examp), we show a question from the ANES 2020 questionnaire. This part shows a particular question's question name (`postvote_rvote`), description (Did R Vote?), full wording of the question and responses, response order, universe, question logic (if `vote_pre` = 0), and other specifications. It also includes the variable name, which can be used to link to the codebook.

```{r}
#| label: que-examp
#| echo: false
#| fig.cap: ANES 2020 Questionnaire Example
#| fig.alt: Question information about the variable postvote_rvote from ANES 2020 questionnaire Survey question, Universe, Logic, Web Spec, Response Order, and Released Variable are included. 

knitr::include_graphics(path="images/questionnaire-example.jpg")
```
````